product/dialogs/miq_dialogs/miq_provision_azure_dialogs_template.yaml

```diff
@@ -0,0 +1,364 @@
+        :owner_city: 
+          :description: City
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_department: 
+          :description: Department
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_load_ldap: 
+          :pressed: 
+            :method: :retrieve_ldap
+          :description: Look Up LDAP Email
+          :required: false
+          :display: :show
+          :data_type: :button
+        :owner_manager_phone: 
+          :description: Phone
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_state: 
+          :description: State
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_office: 
+          :description: Office
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_zip: 
+          :description: Zip code
+          :required: false
+          :display: :hide
+          :data_type: :string
+        :owner_email: 
+          :description: E-Mail
+          :required: true
+          :display: :edit
+          :data_type: :string
+        :request_notes:
+          :description: Notes
+          :required: false
+          :display: :edit
+          :data_type: :string
+      :display: :show
+      :field_order: 
+    :purpose: 
+      :description: Purpose
+      :fields: 
+        :vm_tags: 
+          :required_method: :validate_tags
+          :description: Tags
+          :required: false
+          :options: 
+            :include: []
+
+            :order: []
+
+            :single_select: []
+
+            :exclude: []
+
+          :display: :edit
+          :required_tags: []
+
+          :data_type: :integer
+      :display: :show
+      :field_order: 
+    :environment: 
+      :description: Environment
+      :fields:
+        :placement_auto:
+          :values:
+            false: 0
+            true: 1
+          :description: Choose Automatically
+          :required: false
+          :display: :hide
+          :default: false
+          :data_type: :boolean
+        :resource_group:
+          :values_from:
+            :method: :allowed_resource_groups
+          :description: Resource Groups
+          :required: true
+          :display: :edit
+          :data_type: :integer
+        :cloud_network:
+          :values_from:
+            :method: :allowed_cloud_networks
+          :description: Virtual Private Cloud
+          :auto_select_single: false
+          :default: nil
+          :required: true
+          :display: :edit
+          :data_type: :integer
+        :cloud_subnet:
+          :values_from:
+            :method: :allowed_cloud_subnets
+          :description: Cloud Subnet
+          :required: true
+          :required_method: :validate_cloud_subnet
+          :display: :edit
+          :data_type: :integer
+        :security_groups:
```

Comment:
@gmcculloug - I left this in on purpose because support for security groups will be added next, I'm just waiting on https://github.com/ManageIQ/manageiq/pull/6557 to be pushed through